Tools/parametric_model/configs/quadrotor_model.yaml

```diff
@@ -71,7 +71,7 @@ model_config:
 
 dynamics_model_config:
   optimizer_config:
-    optimizer_class: "QPOptimizer"
+    optimizer_class: "LinearRegressor"
```

Comment:
What is the reason to switch back to the Linear Regressor? This way we will not be able to enforce parameter limits.

Reply:
Sorry. Thanks for pointing that out. 

Indeed the QP optimizer includes limits to eliminate the possibility of unrealistic parameters (e.g. negative friction coefficient) with adding bounds to the problem. The Idea there was that we could highlight that in the feedback to the user to indicate an ill posed data set. It can however be, that the limits in the config file would need to be adapted since we changed quite a lot in the meantime and might have a better idea now of sensible bounds. 

If it comes up with completely wrong parameters we should investigate that further.

Reply:
@manumerous Do we need to address the issue in this PR?

Reply:
Not necessarily I think. But i think it would be good to document the behavior that led to the descision to switch with a reference to this PR in an issue . This way we can keep track of it.

Reply:
I agree. I will open up a separate issue with an example.